pkg/ext-proc/backend/provider.go

```diff
@@ -89,6 +90,15 @@ func (p *Provider) Init(refreshPodsInterval, refreshMetricsInterval time.Duratio
 		}
 	}()
 
+	// Periodically flush prometheus metrics for inference pool
+	flushPrometheusMetricsInterval := 10 * time.Second
```

Comment:
If our recommendation is to have the user scrape every 5s, should we match that here with the flush interval?

Since we want to potentially use this for autoscaling, it might be good to start with a lower value here, especially that we can expect a good amount of volatility here.

Reply:
Make it a flag and default is 5s.